python/python_frontend/python_bindings.cpp

```diff
@@ -4028,6 +4028,67 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::arg("output_gamma") = false,
       py::return_value_policy::reference);
 
+  nvf_ops.def(
+      "cutlass_nvfp4_grouped_mm",
+      [](FusionDefinition::Operators& self,
+         Tensor mat1,
+         Tensor mat2,
+         Tensor scale1,
+         Tensor scale2,
+         Tensor alpha,
+         Tensor problem_sizes,
+         Tensor expert_offsets,
+         Tensor sf_offsets,
+         PrimDataType dtype) -> Tensor {
+        FUSER_PERF_SCOPE("Operators.cutlass_nvfp4_grouped_mm");
+        NVF_CHECK(
+            self.validUse(), "Attempting to add to a completed definition!");
+        FusionDefinition* fd = self.fusion_definition;
+
+        Tensor output = fd->defineTensor(3);
```

Comment:
Doesn't the output tensor have only 2 dimensions or is a third required for k dimension?
```suggestion
        Tensor output = fd->defineTensor(2);
```

Reply:
> is a third required for k dimension?

Yes. We want to keep the reduction dimension in the output, so multi-device can shard on k on inputs and we can support partial reduction on output..